pkg/csi_driver/controller.go

```diff
@@ -259,11 +262,11 @@ func (s *controllerServer) CreateVolume(ctx context.Context, req *csi.CreateVolu
 		if filer.State != "READY" {
 			msg := fmt.Sprintf("Volume %v not ready, current state: %v", name, filer.State)
 			klog.V(4).Infof(msg)
-			return nil, status.Error(codes.Internal, msg)
+			return nil, common.NewTemporaryError(codes.Unavailable, fmt.Errorf(msg))
```

Comment:
in line#260 is DeadlineExceeded a temporary error? if not we should make that temporary too right